app/uk/gov/hmrc/gform/sharedmodel/LocalisedString.scala

```diff
@@ -23,6 +23,8 @@ case class LocalisedString(m: Map[LangADT, String]) {
   def value(implicit l: LangADT): String = m.getOrElse(l, m.getOrElse(LangADT.En, ""))
   def replace(toReplace: String, replaceWith: String): LocalisedString =
     copy(m = (m.map { case (lang, message) => (lang, message.replace(toReplace, replaceWith)) }))
+  def uncapitalize(): LocalisedString =
+    copy(m = m.map { case (lang, message) => (lang, message.splitAt(1) match { case (c, cs) => c.toLowerCase + cs }) })
```

Comment:
Why is it lowercasing only first char? Should it lowercase all text?

Reply:
I might over-complicated things but here is my thoughts
`shortName` of `label` will be used in the middle of the sentence "Select <shortName> from the list."
The form designer might have <short name like "Foo Bar Baz". So I decided to lower case only first letter to make it fit in the middle of the sentence. 
But yes, if we make the whole <shortName> lowercased it would make things easier.
What do you think?

Reply:
This doesn't look right:
```
No match for "North Korie". Select eU country of IOSS registration from the list.
```

Reply:
yes, not sure what to do that. This is not a technical decision here I think.